proposals/0064-remote-control.md

```diff
@@ -0,0 +1,812 @@
+
+
+Things get complicated when passengers are allowed to control. The current implementation allows multiple (two to be exact) devices get connected at the same time. Dy default when a device connects to the system, it is treated as a passenger's device. The driver can change the rank of a device from PASSENGER to DRIVER or vice versa via HMI. SDL-RC allows only one device to be "DRIVER's" at a time. In this document, we refer an application running on a device marked as DRIVER as a driver's application, an application running on a device not marked as DRIVER as a passenger's application.
+
+
+Driver's applications can access/control all available RC modules no matter where the RC module is. Policy rules control what RC modules and how a RC module can be accessed by a passenger's application. Depending on policy configuration, the system can automatically allow, deny or ask driver's permission to grant the access of a passenger's application. 
+
+Unlike a driver's application, which must be launched by the driver on mobile phone and activated on vehicle HMI (HMI level from NONE to FULL), a passenger's application can be launched from a connected phone and automatically activated to HMI level BACKGROUND/LIMITED without HMI touch or voice command after GetIneriorVehicleDataConsent get a successful allowed response (may need driver's permission if configured so). 
+
+By default SDL-RC allows passenger's application to use remote control feature. However, the driver can disable the feature via HMI by sending OnReverseAppsAllowing(false) message to SDL.
+
+Please see attached documents for detailed design. [HMI Guideline] (0064_SDLRC_HMI_Guidelines_v1.1.pdf) and [Mobile API Guideline] (0064_SDLRC_Mobile_API_Guidelines_v1.0.pdf)
+
+### Mobile API changes
+Full Mobile API can be found here:
+https://github.com/smartdevicelink/sdl_core/blob/feature/sdl_rc_functionality/src/components/interfaces/MOBILE_API.xml
+
+The changes are listed below.
+```
```

Comment:
If you can, it would be helpful to break these out into "updated" and "new" sections with the new section broken down further if possible, simply for clarity. This is not a requirement, but would be helpful for others evaluating the proposal.